SETUP.md

```diff
@@ -71,69 +71,64 @@ Finally, flick the repository's switch to enable CI.
-echo https://github.com/$OWNER/$REPO/settings/keys
+echo "https://github.com/$OWNER/$REPO/settings/keys/new"
+
+# For convenience, print the URL to add the private key to Travis CI
+echo "https://travis-ci.com/$OWNER/$REPO/settings"
 ```
 
-Manually add the text of `deploy.key.pub` (with write access) to GitHub under the repository's deploy key settings (the URL echoed above).
-Give the key a descriptive title, such as "Travis CI Manubot".
+#### Add the public key to GitHub
 
-For the next step, you need the [Travis command line client](https://github.com/travis-ci/travis.rb) installed.
-This program is a Ruby gem:
-install it with `gem install travis` (not `apt install travis`, which is a different program).
-After the install, you will need to provide your credentials to login to travis with `travis login`.
+Go to the GitHub settings URL echoed above in a browser, and click "Add deploy key".
+For "Title", add a description like "Manubot Travis Deploy Key".
+Copy and paste the contents of the `deploy.key.pub` text file into the "Key" text box.
+Check the "Allow write access" box below.
+Finally, click "Add key".
 
-```sh
-travis encrypt-file \
-  --com \
-  --repo=$OWNER/$REPO \
-  --force \
-  deploy.key > travis-encrypt-file.log
-```
+#### Add the private key to Travis CI
 
-`ci/travis-encrypt-file.log` will contain a line such as:
+Next, go to the Travis CI repository settings page (URL echoed above).
+Add a new record in the "Environment Variables" section.
+For "NAME", enter `MANUBOT_SSH_PRIVATE_KEY`.
+Next we will encode the text in `deploy.key` to remove newlines, which is required for entry into the Travis settings.
+Run any of the following commands (whichever works), and copy/paste the output into "VALUE":
 
-```
-openssl aes-256-cbc -K $encrypted_8cc93e35fb85_key -iv $encrypted_8cc93e35fb85_iv -in test.key.enc -out test.key -d
-```
+```shell
+# For systems with Python (2 or 3) installed
+python -c "import base64; print(base64.standard_b64encode(open('deploy.key', 'rb').read()).decode())"
 
-`ci/deploy.sh` must be updated with the random string from this line (i.e. `8cc93e35fb85`).
-The following command automates this substitution.
+# For Windows or Linux systems that have the GNU coreutils base64 command
+base64 --wrap=1000000 deploy.key
 
-```sh
-# Edit ci/deploy.sh with travis secure env variables generated by travis encrypt-file
-TRAVIS_ENCRYPT_ID=`python -c "import re; \
-  text = open('travis-encrypt-file.log').read(); \
-  print(re.search('encrypted_([a-f0-9]+)_key', text).group(1))"`
-sed "s/9befd6eddffe/$TRAVIS_ENCRYPT_ID/g" deploy.sh > tmp && mv -f tmp deploy.sh
+# For macOS systems
+base64 --break=1000000 deploy.key
 ```
 
-Next, limit [concurrent](https://blog.travis-ci.com/2014-07-18-per-repository-concurrency-setting/) Travis CI jobs to ensure previous builds deploy before subsequent ones begin:
+Make sure "Display value in build logs" remains toggled off (the default).
```

Comment:
I'm thinking of adding a screenshot of the Travis settings page here (once we finalize the variable name).

Reply:
A screenshot will be a great idea.  Are there additional screenshots we should add for other steps?

Reply:
Let's save screenshots for a subsequent PR.